src/style.css

```diff
@@ -1 +1,39 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('images/star-active.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star {
+  background-image: url('images/star.svg');
+}
+
+.stars--1 .stars__star:nth-child(n + 2) {
+  background-image: url('images/star.svg');
+}
+
+.stars--2 .stars__star:nth-child(n + 3) {
+  background-image: url('images/star.svg');
+}
+
+.stars--3 .stars__star:nth-child(n + 4) {
+  background-image: url('images/star.svg');
+}
+
+.stars--4 .stars__star:nth-child(n + 5) {
+  background-image: url('images/star.svg');
+}
```

Comment:
This is a great implementation, but as a suggestion, the default behavior of stars should be inactive and adding the --N modifiers should activate the N first children of it, can you think of a way to invert your solution to follow this logic?

This is an important change (though not requested by the exercise, this is actually more of a personal suggestion) to improve your overall code maintainability and readability.
In this way it is way more complicated to understand what exactly the modifier does, but most importantly, imagine you now has to implement this logic for 10 stars; with this approach you'll have to change every single one of the classes to make sure it works as intended, the opposite approach would be able to reuse all the previous defined classes and simply add the remaining ones;

This is already a very long comment, if you have any questions please ask them in the qna chat